ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/user-notifications/user-notifications.js

```diff
@@ -25,10 +25,9 @@ const Empty = styled.div`
   transition: transform ${props => props.theme.coreTransitionTime} linear;
   transform: scale(1);
   text-align: center;
-  font-size: ${props => props.theme.largeFontSize}
-  padding: ${props => props.theme.mediumSpacing}
+  font-size: ${props => props.theme.largeFontSize};
```

Comment:
@andrewkfiedler 	@djblue 	Is there a style rule we can add to catch issues like this?

Reply:
There's supposed to be one, not sure why it isn't catching these sorts of things.

Reply:
Should that rule be fixed as part of this change?

Reply:
No, but I think we should open an issue and look into it.